db/seeds.rb

```diff
@@ -6,8 +6,16 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-Lifehack.create(
-  [{ title: 'How to tie a tie',
-    description: 'tie a knot' },
-    { title: 'How to tie a tie1' }]
+User.create([{
+  first_name: 'Jon',
+  last_name: 'Knot',
+  email: 'jon@gmail.com',
+  password: 'whattheheck?' }]
+)
+Lifehack.create([{
+  title: 'How to tie a tie',
+  description: 'tie a knot',
+  creator_id: "#{User.first.id}" }, {
+  title: 'How to tie a tie1',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.